modules/network/src/main/java/org/apache/ignite/internal/network/configuration/KeyStoreType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.network.configuration;
+
+import java.util.Arrays;
+import org.jetbrains.annotations.Nullable;
+
+/** Supported key store types. */
+public enum KeyStoreType {
+    JKS, PKCS12;
```

Comment:
I don't think we even should restrict the list of keystore types. The user might want to add his own keystore type and implementation and we should support such scenarios.

Reply:
I would prefer to remove them when we actually will be going to support that feature

Reply:
removed the whole class